h2o-py/tests/testdir_algos/grid/pyunit_grid_manual_export_resume_new_hyperspace.py

```diff
@@ -0,0 +1,53 @@
+import sys, os
+import tempfile
+
+sys.path.insert(1, os.path.join("..", "..", ".."))
+import h2o
+from tests import pyunit_utils
+from collections import OrderedDict
+from h2o.grid.grid_search import H2OGridSearch
+from h2o.estimators.gbm import H2OGradientBoostingEstimator
+from shutil import rmtree
+
+
+def grid_resume():
+    train = h2o.import_file(path=pyunit_utils.locate("smalldata/iris/iris_wheader.csv"))
+
+    # Run GBM Grid Search
+    ntrees_opts = [1, 3]
+    learn_rate_opts = [0.1, .05]
+    hyper_parameters = OrderedDict()
+    hyper_parameters["learn_rate"] = learn_rate_opts
+    hyper_parameters["ntrees"] = ntrees_opts
+    print("GBM grid with the following hyper_parameters:", hyper_parameters)
+    
+    export_dir = tempfile.mkdtemp()
```

Comment:
files shouldn't be created in temp directory - use test sandbox directory instead

Reply:
Swapped for `pyunit_utils.locate("results")`. Thanks.